src/modules/m_ircops.cpp

```diff
@@ -0,0 +1,131 @@
+
+/*
+ * InspIRCd -- Internet Relay Chat Daemon
+ *
+ *   Copyright (C) 2015 Antonio Costa <bigua@brasirc.com.br>
+ *   Copyright (C) 2014 Joseph Newing <jnewing@gmail.com>
+ *
+ * This file is part of InspIRCd.  InspIRCd is free software: you can
+ * redistribute it and/or modify it under the terms of the GNU General Public
+ * License as published by the Free Software Foundation, version 2.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * *
+ * m_ircops.cpp  - ver1.1
+ *
+ *  This module was been update to core 2.0 the command allow also print out
+ *  a list of current aways opers.
+ *
+ */
+
+
+/* *
+ * m_ircops.cpp - ver 1.0
+ *
+ * Module adds the IRCOPS command that prints out a list of current online
+ * opers including their class unless they have mode +H set on themselves.
+ *
+ * NOTES:
+ * THIS COMMAND IS NOT LIMITED TO OPERS!
+ *
+ * I tried to keep the formatting of the output messages as close as i could
+ * to kind of fit the InspIRCd look (it looks a lot like the CHECK command)
+ *
+ */
+
+/* $ModDesc: /IRCOPS command that prints out a list of current online */
+/* $ModAuthor: Joseph Newing (synmuffin) */
+/* $ModAuthorMail: jnewing@gmail.com */
+/* $ModDepends: core 1.2-1.3 */
+
+#include "inspircd.h"
+
+class CommandIRCOps : public Command
+{
+
+public:
+
+    CommandIRCOps (Module* Creator) : Command(Creator,"IRCOPS",0)
+	{
+
+	}
+
+	CmdResult Handle (const std::vector<std::string> &parameters, User *user)
+	{
+		std::string ircops_str;
+		ircops_str = "Total:";
+
+		if (user)
+		{
+			user->WriteServ("============ Online IRC Operators =============");
```

Comment:
You need to give it a command name and prefix the final parameter with a colon.

This will probably not be merged though as it almost entirely duplicates existing core behaviour. You might have better luck over at [inspircd-extras](https://github.com/inspircd/inspircd-extras) but even then i'm not sure if it would be merged.